### DISCUSSION TITLE
post/seven-months-of-struggle-preface

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#12] [General]
# post/seven-months-of-struggle-preface

PS：真的真的很抱歉拖了这么久才发出来，最近实习事情也不少，我尽量恢复到之前的更新频率吧～  七个月的挣扎之路终于结束了，这是我从未承受过的痛苦 具体来说，是从去年 2023 年 8 月 22 日返校开始，再到 2024 年 4 月 3 日 腾讯实习 Offer Call 结束，过去了七个月有余

https://nickxu.me/post/seven-months-of-struggle-preface

**Comment by HuanXin-Chen** [comment]
加油学长，你很棒的！（在你的文章中获得很多共鸣）

**Comment by psyq55262227** [comment]
不得不说，其实华农不是211，而且计算机也不算强势学科。我生源地也是广东，而且是垫底的分数进来的，感觉能来杭电的这个分数在广东是比较尴尬的（）虽然我也经常吐槽杭电，但它确实是从当时来看这个分数段可以选的、计算机氛围比较好的学校了。能再选一次的话，把圣光机从志愿里去掉就比较完美了x

**Comment by YXZ252426** [comment]
学长，你真的是我的偶像，我现在是杭电计算机大二的，我的成长路线和你也有些类似，参加过一次青训营真的是让我收获很多，我目前也打算争取一份字节的实习，你的文章对我有非常大的帮助

**Comment by illyber** [comment]
你的执行力真的强。
